frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1231,18 +1231,25 @@ class Worksheet extends React.Component {
                 aria-labelledby='delete-worksheet-confirmation-title'
                 aria-describedby='delete-worksheet-confirmation-description'
             >
-                <DialogTitle id='delete-worksheet-confirmation-title'>
-                    {'Warning: Deleted worksheets cannot be recovered'}
+                <DialogTitle id='delete-worksheet-confirmation-title' style={{ color: 'red' }}>
+                    Delete this worksheet permanently?
                 </DialogTitle>
                 <DialogContent>
+                    <DialogContentText id='alert-dialog-description' style={{ color: 'red' }}>
+                        {'Warning: Deleted worksheets cannot be recovered'}
+                    </DialogContentText>
                     <DialogContentText id='alert-dialog-description' style={{ color: 'grey' }}>
                         {'Note: deleting worksheets does not delete the bundles inside it.'}
```

Comment:
it may be nice to have a "force delete" option for worksheets, but that should probably be a separate issue

Reply:
I don't think deleting worksheet would be blocked by anything though, deleting worksheet doesn't delete the bundles inside